kalliope/core/OrderAnalyser.py

```diff
@@ -56,6 +58,9 @@ def get_matching_synapse(cls, order, brain=None):
         # create a list of MatchedSynapse from the tuple list
         list_synapse_to_process = cls.get_list_synapses_to_process(list_match_synapse)
 
+        # save in kalliope memory the last stt order
+        Cortex.save("kalliope_last_stt_message", order)
```

Comment:
I would prefer "last_order" for this one. Also, you need to update the doc.

Reply:
Key name changed.
kalliope_last_order is already documented, what do you mean?

Reply:
last tts message is documented. But what you've just added is not, doesn't it?